go.mod

```diff
@@ -336,4 +337,4 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
-replace github.com/cosmos/cosmos-sdk => github.com/bryanchriswhite/cosmos-sdk v0.50.1-fix-sequencer-check
```

Comment:
dis is important https://github.com/bryanchriswhite/cosmos-sdk/compare/v0.50.1-fix-sequencer-check...okdas:cosmos-sdk:v0.50.1-fix-sequencer-check-2